react-basic-practice/src/components/Modal/index.tsx

```diff
@@ -6,14 +6,17 @@ import {
   ModalHeader,
   ModalFooter,
   ModalBody,
+  Box,
+  Text,
 } from "@chakra-ui/react";
 import Button from "../Button";
 
 interface ModalProps {
   isOpen: boolean;
   onClose: () => void;
   title: string;
-  isEdit?: boolean;
+  isConfirmDelete?: boolean;
```

Comment:
It should be `isConfirm`. We can use it to confirm something else. Not only to delete

Reply:
i done